clients/da-vinci-client/src/main/java/com/linkedin/davinci/DaVinciBackend.java

```diff
@@ -518,24 +471,6 @@ final synchronized void bootstrap() {
             blobTransferManager,
             configLoader.getVeniceServerConfig());
     ingestionBackend.addIngestionNotifier(ingestionListener);
```

Comment:
nit:  After this cleanup, the bootstrap() method lost most of its meaning, can consider renaming or merging with the storage engine check function.